app/models/canvas.rb

```diff
@@ -0,0 +1,13 @@
+class Canvas < ActiveRecord::Base
+  has_many :annotation_lists
+  has_many :annotations, through: :annotation_lists
```

Comment:
Again, doesn't this collide with the `annotations` method below?

Reply:
I believe the original rationale was to see if it would automatically return all layers and make it unique, but it didn't. I will delete!